src/modules/tests/tests-reducer-factory.ts

```diff
@@ -0,0 +1,20 @@
+import { TestCategory } from '../../shared/models/test-category';
+import { Action } from '@ngrx/store';
+import { StandardCarTestCATBSchema } from '@dvsa/mes-test-schema/categories/B';
+import { StandardTrailerTestCATBESchema } from '@dvsa/mes-test-schema/categories/BE';
+import { testsCatBReducer } from './tests.cat-b.reducer';
+import { testsCatBeReducer } from './tests.cat-be.reducer';
+
+type unionType = Required<StandardCarTestCATBSchema> | Required<StandardTrailerTestCATBESchema>;
```

Comment:
Really nice. However I think unionType should be something more meaningful like schemaUnionType